### PR TITLE
Setup auto-reloading 

### PR DESCRIPTION
**Pull request by daltonfury42**
I was checking out this tool which will auto rebuild on source file changes while developing locally.

https://github.com/cosmtrek/air

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,10 +1,17 @@
-all: AzureFunctions/handler
+azure: AzureFunctions/handler
```
Comment (daltonfury42) [inline Makefile]: I've not worked with makefiles before, I might have gotten this wrong, let me read make properly.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+1. Golang setup: https://go.dev/doc/install
+2. Start development server, 
+```
+git clone git@github.com:SimplQ/simplQ-golang.git
+cd simplQ-golang/
+air
+```
+
+We use Visual Studio Code as the IDE but this is not a strict requirement. [Azure Functions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) plugin is recommended if you was to deploy code to Azure, but this is optional.
```
Comment (maaverik) [inline README.md]: Nit: if you "want" to deploy

File: README.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+plugin is used to deploy code.
```
Comment (maaverik) [inline README.md]: Nit: plugin?

**Review by maaverik — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# simplQ-golang
+
+## Local Setup Notes
+
+1. Golang setup: https://go.dev/doc/install
+2. Start development serverL
+```
```
Comment (maaverik) [inline README.md:7]: Nit: server

**Review by maaverik — APPROVED** [review]

**Pull request merged**